### DISCUSSION TITLE
Job Opportunity - University of Florida Insect Biodiversity Data Manager

### DISCUSSION THREAD

**Discussion by ForestEntomologyLab** [Symbiota/community-board#586] [General]
The UF Forest Entomology Lab is looking for an Insect Diversity Data Manager. The ideal applicant will have experience with relational and other databases, biodiversity data standards, and the science of taxonomy and systematics.

Details:

- Annual salary: $50,000 including full health benefits.
- Duration: This will be a one-year position with a high likelihood of continuation depending on performance. The contract will include three months of probation period.
- Position start: flexible, ideally before the end of 2024.
- In-person work is preferred. Remote work is possible, but at least occasional physical presence in Gainesville, FL, and/or Austin, TX, will be required.

To apply, send the following to ForestEntomology@ifas.ufl.edu

- A letter describing your experience with entomology, biosystematics, or related fields.
- Your CV
- A list of your hobbies (what do you do when nobody tells you what to do?)
- Application deadline: September 09, 2024.

For the full job posting, click here: https://pitchtube.org/job-opportunity-insect-biodiversity-data-manager/

**Comment by themerekat** [comment]
I've sent this to our email list, but I don't think discussions is the best place for it!

**Reply by ForestEntomologyLab** [reply]
Thank you Katie! Apologies for putting it in the wrong place.

**Reply by themerekat** [reply]
No worries! Happy to help get the info out there.
